index.js

```diff
@@ -230,6 +230,7 @@ module.exports = {
       return async.eachLimit(addDates, 5, function(date, callback) {
         var eventCopy = _.cloneDeep(piece);
         eventCopy._id = self.apos.utils.generateId();
+        eventCopy.workflowGuid = self.apos.utils.generateId();
```

Comment:
Check if the original event has a workflowGuid first, as a cheap verification that workflow is present and this type is subject to workflow.